packages/eslint-config-angular/package.json

```diff
@@ -31,11 +31,11 @@
     "!/.*.js"
   ],
   "peerDependencies": {
-    "@angular-eslint/builder": "^14.0.2",
-    "@angular-eslint/eslint-plugin": "^14.0.2",
-    "@angular-eslint/eslint-plugin-template": "^14.0.2",
-    "@angular-eslint/schematics": "^14.0.2",
-    "@angular-eslint/template-parser": "^14.0.2",
+    "@angular-eslint/builder": "^14.0.2 || ^15.2.0",
```

Comment:
I think I'd vote to change these to `"14.0.0 || 15.0.0"`. That should allow clients to use any minor or patch version of 14 or 15 rather than just versions greater than 14.0.2 or 15.2. We'd still leave the other package file at a specific higher version for testing within this repo.

It's unlikely to make a difference for Skyline clients who will start at 15.2 but it better expresses our intent and might help some other client on an earlier version.

You should rerun `npm install` to regenerate `package-lock.json` after making that change. I doubt it will result in any diffs but it's better to be safe.

Reply:
Yes, my bad, It should have the `^`.

Reply:
I've changed this to `^14.0.0 || ^15.0.0`, and I confirmed that could be installed in SystemLinkShared, which is currently using Angular 14.2.